packages/@aws-cdk/cfnspec/spec-source/cfn-docs/cfn-docs.json

```diff
@@ -10859,7 +10859,7 @@
       },
       "description": "The `AWS::DMS::ReplicationInstance` resource creates an AWS DMS replication instance.",
       "properties": {
-        "AllocatedStorage": "The amount of storage (in gigabytes) to be initially allocated for the replication instance.",
+        "AllocatedStorage": "The amount of storage (in gibibytes) to be initially allocated for the replication instance.",
```

Comment:
We can't change the spec source directly since it is generated and imported from CloudFormation. We can only adjust what's on our L2 API ref. This will need to get fixed upstream by Cfn team